flent/runners.py

```diff
@@ -680,8 +680,17 @@ def find_binary(self, **args):
             args['socket_timeout'] = ""
 
         if args['test'] in ("TCP_STREAM", "TCP_MAERTS", "omni"):
+            self.ctrl_port = args['netperf_control_port']
```

Comment:
Hold on:
Doing from flent

> $ grep -iR NETPERF_CONTROL_PORT *
> flent/settings.py:    action="store", type=int, dest="NETPERF_CONTROL_PORT", metavar="PORT",

You have it as a default in your settings, remember? Checked for other params that are in settings.py. You draw them from args dict as well.